tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -527,6 +527,7 @@ ecmp/test_ecmp_sai_value.py:
     conditions_logical_operator: or
     conditions:
       - "topo_type not in ['t1', 't0']"
+      - "'dualtor' in topo_name"
       - "asic_type not in ['broadcom']"
```

Comment:
Also this condition, we have `pytest.mark.asic('broadcom')` in script

Reply:
Same as the above.